deploy/template.yaml

```diff
@@ -415,14 +415,13 @@ Resources:
               DATA_BTM_SQS_PAYLOAD_EVENTS: "/perfTest/data/txma/SQSPayloadEvents"
               DATA_TXMA_SQS: "/perfTest/data/txma/perfSQS"
               IDENTITY_ADDRESS_URL: "/perfTest/identity/orange/addressUrl"
-              IDENTITY_BUILD_CORE_URL: "/perfTest/identity/build/coreUrl"
               IDENTITY_CIMIT_GETCIC: "/perfTest/identity/cimit/getcic"
               IDENTITY_CIMIT_POSTMITIGATION: "/perfTest/identity/cimit/postmitigation"
               IDENTITY_CIMIT_PUTCI: "/perfTest/identity/cimit/putci"
               IDENTITY_CORE_STUB_PASSWORD: "/perfTest/identity/coreStubPassword"
-              IDENTITY_CORE_STUB_URL: "/perfTest/identity/coreStubUrl"
+              IDENTITY_CORE_STUB_URL: "/perfTest/identity/${ENVIRONMENT}/coreStubUrl"
               IDENTITY_CORE_STUB_USERNAME: "/perfTest/identity/coreStubUsername"
-              IDENTITY_DEV_CORE_URL: "/perfTest/identity/dev/coreUrl"
+              IDENTITY_CORE_URL: "/perfTest/identity/${ENVIRONMENT}/coreUrl"
```

Comment:
These need to be static parameter paths as the Build will fail if any of the parameters were not found

Reply:
Changed params back to static